### PR TITLE
Add images to to rgb.md

### PR DESCRIPTION
**Pull request by troytomasch**
### Description
Closes: #3737 
Added images to illustrate the examples

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by dakshdeepHERE — COMMENTED** [review]
I have an overall comment for you to work on. The image you use needs to replace `an` with `a`. That's it overall it looks good to me.

**Comment by troytomasch** [comment]
> I have an overall comment for you to work on. The image you use needs to replace `an` with `a`. That's it overall it looks good to me.

@dakshdeepHERE Thanks for the feedback. You bring up an interesting point and in my research I have seen it both ways. Some of the articles I saw it written out with a 'an' were https://www.semrush.com/blog/h1-tag/ and https://ahrefs.com/blog/h1-tag/. However, on Codeacademy there is an 'a' before an h2 tag in this article: https://www.codecademy.com/article/glossary-html. I think either gets the point across and I can definitely change it if you feel strongly about it.

**Comment by dakshdeepHERE** [comment]
Hey @troytomasch 👋, you're totally right. If we dig deep into grammar then one might think we use `an` with vowels only but when it comes to using `an` it depends on the pronunciation too. For `h` we kind of pronoun it like `aich` so `an` is right to use it. I suggested you change it because of the other entries present in Codecademy they seem to use `a`.  I just want the entries to be consistent. Let's change it for now and wait for the second review maintainer to come and check the change. I'm amazed you did the checks from your side too. Thanks for contributing this entry. 🚀

**Comment by troytomasch** [comment]
@dakshdeepHERE I agree that maintaining consistency should be the priority. I'll make that change. Thanks for the feedback!

**Comment by dakshdeepHERE** [comment]
@troytomasch Thanks for the changes. This looks good for a second review.

**Review by dakshdeepHERE — APPROVED** [review]

File: content/css/concepts/colors/terms/rgb/rgb.md
Comment (caupolicandiaz) [inline content/css/concepts/colors/terms/rgb/rgb.md:39]:
```suggestion
![RGB White Font Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-rgb-example1.png)
```

Comment (caupolicandiaz) [reply content/css/concepts/colors/terms/rgb/rgb.md:39]: The files are in the wrong location. Please relocate the files to the filepath noted above

File: content/css/concepts/colors/terms/rgb/rgb.md
Comment (caupolicandiaz) [inline content/css/concepts/colors/terms/rgb/rgb.md:59]: This color is very light and is difficult to see. I would recommend changing the color and the image to something like rgb(124, 124, 242)

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@troytomasch I've added my comments. Please update the entry, and tag me when you've made the changes. Thanks

**Comment by troytomasch** [comment]
@caupolicandiaz Made the changes you requested. Currently the images show as not found however as they have not been added. Is this the expected behavior?

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz Made the changes you requested. Currently the images show as not found however as they have not been added. Is this the expected behavior?

Yes, the images will not show in the preview.

**Review by caupolicandiaz — APPROVED** [review]
@troytomasch This is ready to merge

**Pull request merged**
